.github/workflows/docker_publish.yml

```diff
@@ -14,28 +14,29 @@ jobs:
 
     strategy:
       matrix:
-        ubuntu_versions : [
+        ubuntu_version : [
           20.04,
           22.04,
         ]
         compiler : [
           gcc,
           clang,
         ]
-        hdf5_versions : [
+        hdf5_version : [
           1.14.3,
         ]
-        moab_versions : [
+        moab_version : [
           5.4.1,
           5.5.1,
         ]
-        double_down : [
-          OFF,
-        ]
-        geant_version : [
+        geant4_version : [
           10.7.4,
           11.1.2
         ]
+        double_down_version : [
+          v1.0.0,
```

Comment:
We know this version fails, so we should remove it.

We may also want to add `off` to this list so we can test without DD.

Reply:
I have added this to check if CMake 3.16 shows any error messages since it is lower than 1.1.0, but it just passed. That means, CMake will also get failed for Geant4 as well.